tests/metadata_test.py

```diff
@@ -0,0 +1,103 @@
+"""Tests for metadata.py"""
+
+import string
+import random
+import tempfile
+import numpy as np
+import pytest
+
+from git_theta import metadata
+from tests.testing_utils import (
+    random_lfs_metadata,
+    random_tensor_metadata,
+    random_theta_metadata,
+    random_param_metadata,
+    random_metadata,
+)
+
+
+def metadata_equal(m1, m2):
+    m1_flat = m1.flatten()
+    m2_flat = m2.flatten()
+    if m1_flat.keys() != m2_flat.keys():
+        return False
+    for k in m1_flat.keys():
+        if m1_flat[k] != m2_flat[k]:
+            return False
+    return True
+
+
+def test_lfs_pointer():
+    """
+    Test LfsMetadata creates and reads LFS pointers correctly
+    """
+    version = "my_version"
```

Comment:
Are `version` `oid` and `size` unused or am I missing something?

Reply:
Yup left it in by mistake.